Sources/ComposableArchitecture/Reducer/ReducerBuilder.swift

```diff
@@ -115,7 +115,7 @@ public enum ReducerBuilder<State, Action> {
     @usableFromInline
     let wrapped: Wrapped?
 
-    @inlinable
+    @usableFromInline
```

Comment:
I think we want to annotate with both if possible. Since `@usableFromInline` calls are not unlined unless they also contain the other annotation. Wanna try adding both and seeing if things still build?

Reply:
Unfortunately not. Adding both produces a warning in regular builds and the same error as before in library evolution builds.